common/djangoapps/track/middleware.py

```diff
@@ -70,7 +83,12 @@ def process_request(self, request):
 
             views.server_track(request, request.META['PATH_INFO'], event)
         except:
-            pass
+            ## Why do we have the overly broad except?
+            ##
+            ## Adding instrumentation. If we're dropping events on the floor,
+            ## we should know about it.
+            event = {'event-type': 'exception', 'exception': sys.exc_info()[0]}
```

Comment:
I'm not sure the exception detail is terribly useful in this case. I think I might remove it from the tracking log event and instead call log.exception() to ensure it ends up in standard logs.